Cargo.toml

```diff
@@ -184,7 +184,8 @@ ethers-signers = { version = "2.0", default-features = false }
 ethers-middleware = { version = "2.0", default-features = false }
 
 discv5 = { git = "https://github.com/sigp/discv5", rev = "f289bbd4c57d499bb1bdb393af3c249600a1c662" }
-igd = { git = "https://github.com/stevefan1999-personal/rust-igd", rev = "c2d1f83eb1612a462962453cb0703bc93258b173" }
+# Fork of rust-igd with ipv6 support
+igd-next = { git = "https://github.com/dariusc93/rust-igd", rev = "2b268064c70e62cfea26952436630ea898d3b922" }
```

Comment:
can we use this from crates instead? looks lik it's up to date

https://crates.io/crates/igd-next

Reply:
actually yes that's better, did not notice that it's on crates